packages/terra-core-docs/src/terra-dev-site/doc/toolbar/About.1.doc.mdx

```diff
@@ -40,6 +43,9 @@ import Toolbar from 'terra-toolbar';
 
  ## Examples
  <DefaultToolbar />
+ <AriaLabelToolbar />
+ <AriaLabelledByToolbar />
+ <AriaControlsToolbar />
```

Comment:
Adds a more descriptive title and description sentence:

```suggestion
 <DefaultToolbar title="A Basic Toolbar using the aria-label and aria-controls attributes for accessibility" 
 description="For assistive technology users, this example adds the aria-label attribute to name the type tools this 
 toolbar contains, and adds the aria-controls attribute to indicate which page region the toolbar applies to." />
```

Reply:
Updated [d92fdde](https://github.com/cerner/terra-core/pull/3678/commits/d92fdde1469050de5ac17f97563b115a8d4162a4)